capsule/src/SessionManager.ts

```diff
@@ -1,29 +1,100 @@
 import userManagementClient from './UserManagementClient';
-import { SessionStorage } from './SessionStorage';
+import {SessionStorage} from './SessionStorage';
+import {
+  PublicKeyStatus,
+  PublicKeyType,
+} from '@usecapsule/user-management-client';
+import {InAppBrowser} from 'react-native-inappbrowser-reborn';
+import {
+  getWebAuthURLForCreate,
+  getWebAuthURLForLogin,
+  isSessionActive,
+} from './helpers';
 
 export default class SessionManager {
   private userId: string;
   private sessionStorage: SessionStorage;
+  private webauth: boolean;
   public async setSessionKey() {
-    return await userManagementClient.addSessionPublicKey(this.userId, {
-      publicKey: await this.sessionStorage.getPublicKey(),
-    });
+    const res = await userManagementClient.addSessionPublicKey(
+      this.userId,
+      this.webauth
+        ? {
+            status: PublicKeyStatus.PENDING,
+            type: PublicKeyType.WEB,
+          }
+        : {
+            publicKey: await this.sessionStorage.getPublicKey(),
+          }
```

Comment:
```suggestion
          },
```

See Google style guide 5.2.1 Use trailing commas